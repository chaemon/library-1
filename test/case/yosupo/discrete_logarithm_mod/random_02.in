88
2028037 288974909 451794315
22868176 21545984 111681589
164199418 12017458 177741458
727272894 383855815 745629016
263419161 317155013 451963118
442332844 275132246 625817845
239303569 23523747 358996391
142493592 158426858 398318282
7061287 14371411 499848368
79384002 19141560 622426889
313082157 130997331 513456961
120030523 450051619 463541290
96010572 7760920 112618851
142464198 334404235 504459523
674397136 404243924 867470926
81191004 58283186 86377175
435167767 71459492 791513592
52898442 257607102 341562699
169454058 184051746 364178764
112784412 626625485 653545260
132438830 81549261 423006840
63159789 132602785 342823401
78145328 227723599 552010352
128456734 26943129 266280592
15373180 223178710 530649786
34180782 13453515 285284865
39450812 27279851 172354130
711708323 241389523 835825144
95282135 388198169 601591619
122021473 420267989 895143572
207534299 532166635 870020618
66284175 123932736 503056551
118884818 169564469 200677820
41670466 35709788 66950057
83961528 91616037 175415880
446057125 389131933 510176862
330245041 84787112 368100056
231622253 121236313 257571541
167265452 399008442 864636174
13976582 458109739 497916597
568056529 253805046 758608227
433527490 86561732 617776916
330922633 396169942 438782171
277614808 316185667 448906709
597214909 510226327 626274618
180426754 43412191 461020002
542275936 209267734 868977855
118423689 84098970 120714503
161552223 119634118 314300241
4622329 30100233 30107400
123862180 160906495 549763655
35878171 29284385 46147266
260015621 152536706 294403015
114015626 46195566 121038460
474964155 587182878 721753846
433177566 195162271 550621782
910191630 114548429 969391499
64884460 459208762 474202087
364830411 552865817 935590237
560792314 523106616 688217588
152225730 99227632 739024886
59551364 50610948 423647481
362895767 204797314 689797243
97223158 322778310 532352219
14371406 327962270 715453292
434672391 105983829 842741244
77198629 271393373 375024121
745140973 56268139 996866915
78552171 111933668 759836601
28006754 195253020 204658093
672712383 468924696 688293140
338013330 700743486 777760360
180352086 336664642 608345216
168859884 39482319 280134952
310076619 304003094 761089113
39345059 73479089 83056179
819145862 859516181 863471205
205980989 72726621 701455875
17373796 45856148 87302528
6608577 464022604 582435992
146395479 162107255 175084791
80382662 288197137 484718005
343677205 285797128 495045359
905111292 478112150 975435311
226665179 114673431 284212394
389190500 108514522 807004527
73411128 346897 78431293
281288305 339577893 793108761
