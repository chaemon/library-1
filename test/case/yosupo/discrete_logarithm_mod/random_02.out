-1
-1
-1
-1
74988421
-1
-1
-1
-1
22512482
-1
-1
-1
3916597
12633090
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
6104912
-1
-1
-1
17682731
-1
775817034
-1
55993738
-1
23900792
203139840
-1
-1
-1
5835698
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
1375103
-1
54099585
-1
